src/virtual-wan/azext_vwan/custom.py

```diff
@@ -334,16 +361,16 @@ def update_vhub_route_table(cmd, resource_group_name, virtual_hub_name, route_ta
         client = _v3_route_table_client(cmd.cli_ctx)
         route_table.labels = labels
 
-    return sdk_no_wait(no_wait, client.create_or_update, resource_group_name,
+    return sdk_no_wait(no_wait, client.begin_create_or_update, resource_group_name,
                        virtual_hub_name, route_table_name, route_table)
 
 
 def get_vhub_route_table(cmd, resource_group_name, virtual_hub_name, route_table_name):
-    from msrestazure.azure_exceptions import CloudError
+    from azure.core.exceptions import ResourceNotFoundError
     try:
         return _v3_route_table_client(cmd.cli_ctx)\
             .get(resource_group_name, virtual_hub_name, route_table_name)  # Get v3 route table first.
-    except CloudError as ex:
+    except ResourceNotFoundError as ex:
```

Comment:
>ResourceNotFoundError [](http://example.com/codeflow?start=11&length=21)

`ResourceNotFoundError` already means `ex.status_code == 404`.

The original logic is we won't raise exception only when status_code is 404; other cases we should raise exception.

Use following code is enough,

```Python
except ResourceNotFoundError as ex:
            return _v2_route_table_client(cmd.cli_ctx)\
                .get(resource_group_name, virtual_hub_name, route_table_name)  # Get v2 route table.
```